www/libhttpseverywhere/Portfile

```diff
@@ -23,14 +23,29 @@ long_description    ${description}. HTTPSEverywhere is a browser plugin \
                     GLib-based library you can link/bind against \
                     in almost all languages
 
-homepage            https://git.gnome.org/browse/${name}
-master_sites        gnome:sources/${name}/${branch}/
-
 use_xz              yes
 
-checksums           rmd160  67ecbcfbfe5eba4eba47b38099ddc4ebae847403 \
+set main_distfile   ${distfiles}
+set rules_distfile  default.rulesets-2019.1.7
+set rules_commit    9c43785fe5ce13c70b8a1abfece834d12fc9d0fb
+
+extract.only        ${main_distfile}
+
+distfiles           ${main_distfile}:main \
+                    ${rules_distfile}:rules
+
+homepage            https://git.gnome.org/browse/${name}
+master_sites        gnome:sources/${name}/${branch}/:main \
+                    'https://gitlab.gnome.org/GNOME/libhttpseverywhere/-/raw/${rules_commit}/data/default.rulesets?inline=false&dummy=':rules
```

Comment:
You can lose the single quotes here; they don't have any special meaning in Tcl.

Reply:
done.